docs/source/testing/testing.md

```diff
@@ -3,26 +3,55 @@ title: Integration testing
 description: Utilities for testing Apollo Server
 ---
 
-Testing `apollo-server` can be done in many ways. One simple way is to use ApolloServer's `executeOperation` method to directly execute a GraphQL operation without going through a full HTTP operation.
-
-## `executeOperation`
-
-Integration testing a GraphQL server means testing many things. `apollo-server` has a request pipeline that can support many plugins that can affect the way an operation is executed. The `executeOperation` method provides a single hook to run operations through the request pipeline, enabling the most thorough tests possible without starting up an HTTP server.
+Integration testing a GraphQL server means testing the combination of many interlocking parts at once. One way to test your server setup is to use `ApolloServer`'s `executeOperation` method to directly execute a GraphQL operation without going through a full HTTP request.
+
+## Testing using `executeOperation`
+
+The `apollo-server` library has a request pipeline that can support many plugins which can each affect the way an operation is executed. The `executeOperation` method is available to run operations through the request pipeline, enabling the most thorough tests possible without starting up an HTTP server.
```

Comment:
This first legacy sentence was making me 🤔 first time around and I forgot to come back to it. What do you think about this:

Apollo Server uses a multi-step request pipeline to validate and execute incoming GraphQL operations. This pipeline supports integration with custom plugins at each step, which can affect an operation's execution. Because of this, it's important to test the entirety of your request pipeline with a variety of operations. [It's possible this stuff is general-purpose enough that it could go in the _intro_ instead...]

Apollo Server's `executeOperation` method enables you to run operations through the request pipeline _without_ sending an HTTP request.

["multi-step request pipeline" could also link to this plugin section: https://www.apollographql.com/docs/apollo-server/integrations/plugins/#request-lifecycle-event-flow]

Reply:
Yes, I love this! I think I will move "Apollo Server uses a multi-step request pipeline..." to the beginning paragraph to set the scene for what folks will actually be testing with `executeOperation`.